nservicebus/azure-storage-queues/configuration.md

```diff
@@ -60,3 +60,10 @@ This feature can be enabled by specifying `.UseAccountNamesInsteadOfConnectionSt
 snippet:AzureStorageQueueUseAccountNamesInsteadOfConnectionStrings
 
 NOTE: This feature is not available in `NServiceBus.Azure.Transports.WindowsAzureStorageQueues` Versions 6 and below.
+
+## Hashing algorithms
+By default The Azure Storage Queues Transport uses `MD5` hashing algorithm for renaming queues when an entity name is too large. If your solution requires a stronger algorithm, you can enable renaming queues with `SHA1` by specifying `UseSha1ForShortening` when configuring the `AzureStorageQueueTransport`:
```

Comment:
first you say "entity names" then "renaming queues" - it's not clear whether it applies to something other than queues, can you clarify?